modules/planning/scenarios/scenario_manager.cc

```diff
@@ -222,7 +229,42 @@ void ScenarioManager::Observe(const Frame& frame) {
 void ScenarioManager::Update(const common::TrajectoryPoint& ego_point,
                              const Frame& frame) {
   CHECK(!frame.reference_line_info().empty());
+
   Observe(frame);
+
+  if (FLAGS_enable_scenario_dispatcher) {
+    Scenario_Dispatcher(ego_point, frame);
+  } else {
+    Scenario_Voter(ego_point, frame);
+  }
+}
+
+void ScenarioManager::Scenario_Dispatcher(
+    const common::TrajectoryPoint& ego_point,
+    const Frame& frame) {
+  CHECK(!frame.reference_line_info().empty());
+
+  if (current_scenario_->GetStatus() !=
+      Scenario::ScenarioStatus::STATUS_DONE) {
+    ADEBUG << "continue current scenario: " << current_scenario_->Name();
+    return;
+  }
+
+  // change lane case, currently default to LANE_FOLLOW in change lane case.
+  // TODO(all) implement change lane scenario.
+  if (SelectChangeLaneScenario(ego_point, frame)) {
+    ADEBUG << "Use change lane scenario (temporarily use LANE_FOLLOW)";
+    return;
+  }
+
+  // TODO(all): to be added
+}
+
+void ScenarioManager::Scenario_Voter(
+    const common::TrajectoryPoint& ego_point,
```

Comment:
It seems the naming is a bit misleading. It is more like a class name. Change the name to VoteScenario? The same applies to "Scenario_Dispatcher".

Reply:
addressed. thx